src/commands/deploy.rs

```diff
@@ -385,6 +389,18 @@ impl DeployCommand {
                 .await
                 .context("Problem patching a channel")?;
 
+                for kv in self.key_value {
+                    CloudClient::add_key_value_pair(
+                        &client,
+                        app_id,
+                        String::from("default"),
```

Comment:
I'm not sure what to use for store here

Reply:
I think `"default"` is correct for now. However, that is a good point. We may need to provide the option to set the store name when we provide more stores. if we can assume all KVs can be set for a single store, the following may be good: `spin deploy --kv key=value --store "myStore"` where `--store` defaults to "default" if not set. The concern becomes if we want to presetting kv values for multiple stores. At that point it may be better for users to use a CLI or UI. 

Circling back, for the first version of this PR, we should probably just set "default" as a constant since we don't have support for other stores yet. @bacongobbler thoughts?

Reply:
Setting "default" as a constant would be a good first step.

Reply:
Added a constant for default store name